app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/model/VerificationCodeDisplayItem.kt

```diff
@@ -11,8 +11,8 @@ import kotlinx.parcelize.Parcelize
 @Parcelize
 data class VerificationCodeDisplayItem(
     val id: String,
-    val issuer: String?,
-    val label: String?,
+    val title: String,
+    val subtitle: String?,
```

Comment:
Updated UI models (there are two) to use `title` and `subtitle`, to help hopefully make things less confusing. The `title` could be the `issuer` or the `label` depending on nullabilty.